src/hotspot/share/gc/g1/g1_globals.hpp

```diff
@@ -79,7 +79,15 @@
           "The percentage of regions in the collection set starting "       \
           "from the beginning where the forced evacuation failure "         \
           "injection will be applied.")                                     \
-          range(1, 100)
+          range(1, 100)                                                     \
+  product(uint, G1EvacuationFailureALotWorkerCost, 2,                       \
```

Comment:
This has nothing to do with the `EvacuationFailureALot` functionality, but is primarily a weight for the forward pointer removal. This ought to be renamed and moved to the regular flag section.